odoo/addons/base/models/ir_attachment.py

```diff
@@ -341,6 +341,9 @@ def _postprocess_contents(self, values):
                     else:  # datas
                         img = ImageProcess(base64.b64decode(values['datas']), verify_resolution=False)
 
+                    if not img.image:
+                        raise UserError("Empty source, SVG, or WEBP")
```

Comment:
Sorry I did not notice before, but the error message should a translated text.

For the final approval, I do not have such rights.